backend/experiment/migrations/0050_migrate_experiment_content_to_translatedmakemigrations.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 4.2.14 on 2024-07-31 13:41
+
+from django.db import migrations
+
+
+def migrate_experiment_content(apps, schema_editor):
+    Experiment = apps.get_model("experiment", "Experiment")
+    ExperimentTranslatedContent = apps.get_model("experiment", "ExperimentTranslatedContent")
+
+    for experiment in Experiment.objects.all():
+        ExperimentTranslatedContent.objects.create(
+            experiment=experiment,
+            index=0,
+            language="en",  # Set English as default language
+            name=experiment.name,
+            description=experiment.description,
+            consent=experiment.consent,
+            about_content=experiment.about_content,
+        )
+
+
+def reverse_migrate_experiment_content(apps, schema_editor):
+    Experiment = apps.get_model("experiment", "Experiment")
+    ExperimentTranslatedContent = apps.get_model("experiment", "ExperimentTranslatedContent")
+
+    for experiment in Experiment.objects.all():
+        # find the English translation or the first one (lowest index)
+        experiment_translations = ExperimentTranslatedContent.objects.filter(experiment=experiment)
+        primary_translation = experiment_translations.order_by("index").first()
+
+        if not primary_translation:
+            continue
+
+        experiment.name = primary_translation.name
+        experiment.description = primary_translation.description
+        experiment.consent = primary_translation.consent
+        experiment.about_content = primary_translation.about_content
+        experiment.save()
+
+    ExperimentTranslatedContent.objects.filter(language="en").delete()
```

Comment:
Why not delete all models in this reverse migration?

Reply:
This was easier. When I was working on migrating Blocks from a FK to a OneToOne I had a lot of issues with migrations, especially if I needed to change something in the migrations.

Reply:
I could still squash or re-create them, now that it probably won't need any changes anymore?

Reply:
I still don't quite understand why it would be easier to just delete the English language objects, instead of deleting all `ExperimentTranslatedContent` objects. (Sorry, I erroneously wrote "models" before, of course you shouldn't just go ahead and delete the models without data migration.)

Reply:
Oh, I see what you mean now. Yeah, deleting all objects would make more sense, thanks!